.github/workflows/e2e.yml

```diff
@@ -143,6 +143,8 @@ jobs:
             class: org.apache.dolphinscheduler.e2e.cases.SqlServerDataSourceE2ETest
           - name: HiveDataSourceE2ETest
             class: org.apache.dolphinscheduler.e2e.cases.HiveDataSourceE2ETest
+          - name: SslShellTaskE2ETest
+            class: org.apache.dolphinscheduler.e2e.cases.ssl.SslShellTaskE2ETest
```

Comment:
This test case is not running normally, the cluster is not started. You should check it.
https://github.com/apache/dolphinscheduler/actions/runs/11365198466?pr=16673

Reply:
> This test case is not running normally, the cluster is not started. You should check it. https://github.com/apache/dolphinscheduler/actions/runs/11365198466?pr=16673

can you tell me where the log of cluster startup exception is? how to determine if a cluster is starting up properly? thank you

Reply:
You can take a look at this. https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-e2e/README.md

Reply:
This is still unaddressed. Don't resolve it.

Reply:
I found that all the checks have been passed, why hasn't this problem been resolved yet

Reply:
`SslShellTaskE2ETest` is not executed at all in `e2e` test cases.